### PR TITLE
docs: add callout for multitenancy workshop

### PR DESCRIPTION
**Pull request by rpaik**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Issue Reference this PR resolves**

[For example #12]

**Description of Changes Made (if issue reference is not provided)**

[Description goes here]

### CODE REVIEW

**Comment by rpaik** [comment]
👋 @hassankhan do we need to add the workshop callout to any other docs pages?

File: docs/content/Configuration/Multitenancy-Setup.mdx [hunk 1]
```diff
@@ -5,6 +5,15 @@ category: Configuration
...
 
+<!-- prettier-ignore-start -->
+[[info |]]
+| The Cube multitenancy workshop is on October 27th at 9-10:30 am PST! If you
+| want to learn why you need multitenancy, how to get started,
+| recipes, you will want to attend this event 😀 <br/> You can register
+| for the workshop at [the event
+| page](https://cube.dev/events/multitenancy/).
+<!-- prettier-ignore-end -->
+
```
Comment (hassankhan) [inline docs/content/Configuration/Multitenancy-Setup.mdx:16]:
```suggestion
<InfoBox>
The Cube multitenancy workshop is on October 27th at 9-10:30 am PST! If you want to learn why you need multitenancy, how to get started, recipes and more, you will want to attend this event 😀 

You can register for the workshop at [the event page](https://cube.dev/events/multitenancy/).
</InfoBox>
```

**Comment by hassankhan** [comment]
@rpaik I was thinking maybe we could add it to the [docs homepage](https://cube.dev/docs/) too?

**Comment by hassankhan** [comment]
Also, do we have a calendar link we can add so users can click the date and add it to their calendar?

**Comment by rpaik** [comment]
> @rpaik I was thinking maybe we could add it to the [docs homepage](https://cube.dev/docs/) too?

Yeah, we can do that do. I'll add it 😄

**Comment by rpaik** [comment]
> Also, do we have a calendar link we can add so users can click the date and add it to their calendar?

People will actually see the "calendar link" when they register (provided by Zoom).

**Pull request merged**